service/Microsoft.Azure.Devices/QueryResponse.cs

```diff
@@ -14,7 +14,7 @@ public class QueryResponse<T> : IEnumerable<T>
     {
         readonly IEnumerable<T> queryResults;
 
-        internal QueryResponse(IEnumerable<T> queryResults, string continuationToken)
+        public QueryResponse(IEnumerable<T> queryResults, string continuationToken)
```

Comment:
This is a public API change (note that we will need to increase the ver number).

Reply:
Can I jump onto this and ask why we are moving it from internal to public?

Reply:
agree to up the version of service client

---
In reply to: [141728780](https://github.com/Azure/azure-iot-sdk-csharp/pull/174#discussion_r141728780) [](ancestors = 141728780)

Reply:
We move it to internal to public because the class is now shared between iothub service client and provisioning service client.  We have plans to refactor these commons into the shared.